python/ray/serve/_private/autoscaling_policy.py

```diff
@@ -123,6 +126,20 @@ def __init__(self, config: AutoscalingConfig):
         # scale_up_periods or scale_down_periods.
         self.decision_counter = 0
 
+    def _log_autoscaling_metrics(
+        self,
+        curr_target_num_replicas: int,
+        current_num_ongoing_requests: List[float],
+        desired_num_replicas: int,
+    ):
+        autoscaling_msg = (
+            "Autoscaling decision:"
+            f"curr_target_num_replicas:  {curr_target_num_replicas}, "
+            f"current_num_ongoing_requests: {current_num_ongoing_requests}, "
+            f"desired_num_replicas: {desired_num_replicas}"
```

Comment:
this should include the deployment name and let's make it more readable:
"Autoscaler updating num_replicas for deployment 'name' from <current> to <target>. current_num_ongoing_requests: ..."